mnist/mnist_vanilla_k8s.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<p>Can we automate the configmap creation with python code so they just have to execute the cell to create the configmap?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/737/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>We haven't made a standardized way to authenicate to DockerHub as part of the vanilla-k8s deployment.</p><p>I feel like it'd be better to addressed at creation or have a standardized way like <code>gcloud configure docker </code> </p><p>So as of now, it's okay for it to be separate?</p>